.travis.yml

```diff
@@ -103,7 +103,29 @@ jobs:
         - scripts/travis/docker.sh rocstreaming/env-ubuntu scripts/travis/linux-runtime-checks/valgrind-debug.sh
         - scripts/travis/docker.sh rocstreaming/env-ubuntu scripts/travis/linux-runtime-checks/valgrind-release.sh
 
-    - name: "Android"
+    - name: "Android x86"
+      stage: Build
+      os: linux
+      dist: bionic
+      sudo: required
+      services:
+        - docker
+      before_install:
+        - docker pull rocstreaming/env-android
+        - sudo apt-get update
+        - sudo apt-get install -y qemu-kvm libvirt-bin bridge-utils virtinst virt-manager
+        - sudo adduser $USER libvirt
+        - sudo adduser $USER kvm
```

Comment:
Why do we need kvm on travis if we're running tests inside docker?

Reply:
kvm must be available on host system.
I was running tests on travis beta platform and sometimes hardware acceleration wasn't available so it was needed to install kvm explicitly; if hardware acceleration is always available on travis stable platform we can safely remove these lines.

Reply:
Oh, I see. I don't know. If you're not sure, let's keep it.

Reply:
It seems that kvm is always available on travis stable platform so I've removed explicit kvm installation.